src/asserts.ts

```diff
@@ -0,0 +1,53 @@
+export function assertString(v: unknown): asserts v is string {
```

Comment:
I ended up writing these by hand instead of using zod. zod is just really large

Reply:
Too bad, I tried zod out, it is pretty nice. What about `runtypes` ?